docs/modules/user-guide/partials/proc_devfile-registry.adoc

```diff
@@ -0,0 +1,45 @@
+[id="proc_devfile-registry_{context}"]
```

Comment:
* Don't add the module type to the ID (regardless of what the current templates show).
* This is not a procedure module. The file needs to be renamed, and the Antora page fixed.

```suggestion
[id="devfile-registry_{context}"]
```